tsconfig.json

```diff
@@ -19,7 +19,6 @@
         "noPropertyAccessFromIndexSignature": true,
         "noResolve": false,
         "noUncheckedIndexedAccess": true,
-        "noUnusedLocals": true,
```

Comment:
We could use the linter for this purpose :)

Reply:
yeah I agree